packages/material-ui/src/Snackbar/Snackbar.js

```diff
@@ -98,7 +98,7 @@ export const styles = (theme) => {
 const Snackbar = React.forwardRef(function Snackbar(props, ref) {
   const {
     action,
-    anchorOrigin: { vertical, horizontal } = { vertical: 'bottom', horizontal: 'center' },
+    anchorOrigin: { vertical, horizontal } = { vertical: 'bottom', horizontal: 'left' },
     autoHideDuration = null,
```

Comment:
>The notification now displays at the bottom left on large screens.
  It better matches the behavior of Gmail, Google Keep, material.io, etc.
  You can restore the previous behavior with:

the proposal means this on migration.md ? I think this would be good.

changing `Snackbar` like this diff changes the position on small screen too, that is not what we meant right?

Reply:
> but just changing Snackbar like this diff changes the position on small screen too, that is not what we meant right?

On mobile, the horizontal alignment is ignored, it's full width. So the change has no impact. Maybe we could mention it in the description of the prop?

Reply:
> On mobile, the horizontal alignment is ignored,

Oh, yes it is! 
covering both case would be nice

Reply:
More like this, we don't care about the default value of the prop, what matter is the CSS behavior.

```diff
  /**
  * The anchor of the `Snackbar`.
+ * On smaller screens, the component grows to occupy all the available width,
+ * the horizontal alignment is ignored.
  */
  anchorOrigin?: SnackbarOrigin;
``

Reply:
Thanks a lot.

I committed the update https://github.com/mui-org/material-ui/pull/21980/commits/6c4aece32d84631bd347ee8827f5782552331fc7